include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
BTW, to workaround warning in fi_log construct may be used:
`#if defined ENABLE_DEBUG && ENABLE_DEBUG > 0`